tensorboard/webapp/runs/store/runs_reducers_test.ts

```diff
@@ -965,5 +966,67 @@ describe('runs_reducers', () => {
         ])
       );
     });
+
+    it('sets initial groupby to EXPERIMENT on experiment comparion', () => {
+      const state = buildRunsState({
+        initialGroupBy: {key: GroupByKey.RUN},
+      });
+
+      const nextState = runsReducers.reducers(
+        state,
+        navigated({
+          before: buildRoute({
+            routeKind: RouteKind.EXPERIMENT,
+          }),
+          after: buildCompareRoute(['eid1:run1', 'eid1:run2']),
+        })
+      );
+
+      expect(nextState.data.initialGroupBy.key).toEqual(GroupByKey.EXPERIMENT);
+    });
+
+    it('preserves userSetGroupByKey on comparision view navigation', () => {
```

Comment:
'preserves userSetGroupByKey on comparision view navigation' -> 'preserves userSetGroupByKey set on prior navigation to the same route'